README.md

```diff
@@ -1,104 +1,215 @@
+tqdm
+```
+
+# Modules
+
+## `count`
+
+Calculate TF-IDF information from sequencing data. This module outputs two counting Bloom filters: One for *k*-mer/seed counts (TF) and other for depths (DF). The number of reads will also be printed to stdout.
+
+```
+Usage: count [--help] [--version] [-k VAR] [-s VAR] -f VAR [-e VAR] [-b VAR] [--long] [-t VAR] -o VAR reads
 
 Positional arguments:
   reads          path to sequencing data file(s) [nargs: 0 or more] [required]
 
 Optional arguments:
-  -h, --help     shows help message and exits
-  -v, --version  prints version information and exits
-  -k             k-mer length
-  -s             path to spaced seeds file (one per line, if -k not specified)
+  -h, --help     shows help message and exits 
+  -v, --version  prints version information and exits 
+  -k             k-mer length 
+  -s             path to spaced seeds file (one per line, if -k not specified) 
   -f             path to k-mer spectrum file (from ntCard) [required]
   -e             target output false positive rate [nargs=0..1] [default: 0.0001]
-  -b             output BF/CBF size (bytes)
-  -cmin          minimum count threshold (>=1) [nargs=0..1] [default: 1]
-  -cmax          maximum count threshold (<=255) [nargs=0..1] [default: 255]
-  --counts       output counts (requires ~8x RAM for CBF)
-  --long         optimize for long read data
+  -b             output BF/CBF size (bytes) 
+  --long         optimize for long read data 
   -t             number of threads [nargs=0..1] [default: 1]
   -o             path to store output file [required]
 ```
 
 - Set `-s` to count spaced seeds instead of *k*-mers. `-k` will be ignored in this case. The file pointed to by `-s` should be a plain text file containing spaced seed patterns, one per line, with `1` for care and `0` for don't care positions.
-- [TODO] Use `-w` to find minimizer counts. For each consecutive `-w` *k*-mers/spaced seeds, the element with the smallest hash value represents the window's minimizer.
-- To automatically set `-cmin` to filter erroneous (weak) *k*-mers and find 'solid' ones, use `--solid`. If `--solid` is set, `-cmin` will be ignored.
-- [TODO] Same as `--solid`, use `--repeat` to detect a repeat threshold for `-cmax`.
-- `-f` should point to the histogram file generated by [ntCard](https://github.com/bcgsc/ntCard). Remember to use the same `-k` value for ntCard and ntStat. For spaced seeds, assign the seed length to ntCard's `-k`.
+- `-f` should point to the histogram file generated by [ntCard](https://github.com/bcgsc/ntCard). Use the same `-k` for ntCard and ntStat. For spaced seeds, set ntCard's `-k` parameter to the seed length.
 - Use `-b` to manually set the output data structure's size. Alternatively, set `-e` to the desired false positive rate (0.0001 by default) and ntStat will automatically calculate a suitable value for `-b`.
 
-## `query`
-
-Converts information from a (counting) Bloom filter to TSV format.
-
-### Usage
+## `filter`
```

Comment:
A 1-2 line summary of the various features (i.e., filter, rarity, etc.) would help the reader/user understand your tool better